components/sections/mission/_mission.scss

```diff
@@ -77,14 +77,18 @@ $instagram-embed-bar-height: 66px;
 }
 
 .mission__boomerang {
-  display: inline-block;
+  display: block;
   margin-bottom: 0;
+  margin-left: auto;
+  margin-right: auto;
   max-height: 100vw;
+  max-width: 400px;
```

Comment:
Could be inside an @include

Reply:
with these max-widths, I'm not rem-ing them, because I don't want them to depend on base font size

Reply:
Ok sounds good.